prime-router/src/main/kotlin/azure/ReportQueue.kt

```diff
@@ -158,10 +158,6 @@ object ReportQueue {
         return outputStream.toByteArray()
     }
 
-    private fun createBlobFilename(report: Report): String {
```

Comment:
Half the filename creation was here, and half was in Report.formFileName.   Some code was calling both, some code only calling one.    Consolidated.